docs/en/migration/hook.md

```diff
@@ -1,3 +1,319 @@
 # Migrate Hook from MMCV to MMEngine
 
-Coming soon. Please refer to [chinese documentation](https://mmengine.readthedocs.io/zh_CN/latest/migration/hook.html).
+## Introduction
+
+Due to the development of our architecture design and increasing user demands, existing hook mount points in MMCV are insufficient for the needs. Hence, we redesigned the mount points in MMEngine, and the functions of hooks are adjusted accordingly. Before your migration, it will help a lot to read the tutorial [Hook Design](../design/hook.md).
```

Comment:
```suggestion
Due to the upgrade of our architecture design and increasing user demands, existing hook mount points in MMCV can no logger meet the requirements. Hence, we redesigned the mount points in MMEngine, and the functions of hooks are adjusted accordingly. It will help a lot to read the tutorial [Hook Design](../design/hook.md) before your migration.
```